logdna/resource_stream_config.go

```diff
@@ -0,0 +1,175 @@
+package logdna
```

Comment:
the API for these new resources is consistent, so the separation between `request_types` and `response_types` isn't needed. I kept everything in the one file but can spit it up if preferred.

Reply:
YAY!!! +1